workflows/fastmatchirida.nf

```diff
@@ -148,12 +149,16 @@ workflow FASTMATCH {
     }
 
     // Options related to profile dists
-    mapping_format = Channel.value(params.pd_outfmt)
+    mapping_format = Channel.value("pairwise")
```

Comment:
We are going to 100% of the time use the pairwise format? If so, we could just modify it in the profile_dists command directly. Have less moving parts in the workflow

Reply:
I think so... I didn't want to modify more than necessary for this PR, but I agree with you.